scripts/upload-release-assets.sh

```diff
@@ -45,16 +43,27 @@ REPO="builtin-actors"
 __release_response=`
   curl \
    --header "Authorization: token $GITHUB_TOKEN" \
-   "https://api.github.com/repos/$ORG/$REPO/releases/tags/$release_tag"
+   "$GITHUB_RELEASE_URL"
 `
 __release_id=`echo $__release_response | jq '.id'`
 if [ "$__release_id" = "null" ]; then
-    echo "release $release_tag does not exist"
+    echo "release does not exist"
     exit 1
 fi
 
 __release_upload_url=`echo $__release_response | jq -r '.upload_url' | cut -d'{' -f1`
 
+__release_target_asset=`echo $__release_response | jq -r ".assets | .[] | select(.name == \"$release_target\")"`
+
+if [ -n "$__release_target_asset" ]; then
+    echo "deleting $release_target"
```

Comment:
What happens if the delete fails?  Will the following upload fail as well?

Reply:
A delete failure should fail the script.